soc/xtensa/intel_adsp/common/include/adsp_clk.h

```diff
@@ -61,4 +61,40 @@ struct adsp_cpu_clock_info *adsp_cpu_clocks_get(void);
 #define ADSP_CPU_CLOCK_FREQ_WOVCRO  ADSP_CPU_CLOCK_FREQ(wovcro)
 #endif
 
+
+/* Clock sources used by dai */
+#define ADSP_CLOCK_SOURCE_XTAL_OSC		0
+#if DT_NODE_HAS_STATUS(DT_NODELABEL(audioclk), okay)
+#define ADSP_CLOCK_SOURCE_AUDIO_CARDINAL	1
+#endif
+#if DT_NODE_HAS_STATUS(DT_NODELABEL(pllclk), okay)
+#define ADSP_CLOCK_SOURCE_AUDIO_PLL_FIXED	2
+#endif
+
+#define ADSP_CLOCK_SOURCE_MLCK_INPUT		3
+#ifdef ADSP_CLOCK_HAS_WOVCRO
+#define ADSP_CLOCK_SOURCE_WOV_RING_OSC		4
+#endif
+#define ADSP_CLOCK_SOURCE_COUNT			5
```

Comment:
why do we have the same definitions twice? Can we refactor?

Reply:
Which definition is repeated?

Reply:
see here https://github.com/zephyrproject-rtos/zephyr/pull/57191/commits/7216a367e270a889ec71645b43b20975e443f90e#diff-9a01c394e2ffb65211255e84dd4ada00d8ee88ba0aa20ec31671c27138ea1be9R105